<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>fClimate</class>
 <widget class="QWidget" name="fClimate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>770</width>
    <height>380</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Climate</string>
  </property>
  <widget class="QPushButton" name="pushButton_getdata">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>350</y>
     <width>80</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Get Data</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>281</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Params:</string>
   </property>
   <widget class="QCalendarWidget" name="calendarWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>261</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>220</y>
      <width>261</width>
      <height>101</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QComboBox" name="comboBox_auditories"/>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>MS Shell Dlg 2</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
        </font>
       </property>
       <property name="text">
        <string>Measure:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="comboBox_measures"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Auditory:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <zorder>label_2</zorder>
   <zorder>label</zorder>
   <zorder>comboBox_auditories</zorder>
   <zorder>comboBox_measures</zorder>
   <zorder>calendarWidget</zorder>
   <zorder>calendarWidget</zorder>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>461</width>
     <height>331</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Plot:</string>
   </property>
   <widget class="QCustomPlot" name="customPlot" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>441</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
